packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/TearsheetWrapper.js

```diff
@@ -9,18 +9,18 @@
 
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import CustomizeColumnsModal from './CustomizeColumnsModal';
+import CustomizeColumnsTearsheet from './CustomizeColumnsTearsheet';
 
-const ModalWrapper = ({ instance }) => {
-  const { onSaveColumnPrefs, isModalOpen, setIsModalOpen, labels, ...rest } =
+const TearsheetWrapper = ({ instance }) => {
+  const { onSaveColumnPrefs, isTearsheetOpen, setIsTearsheetOpen, labels, ...rest } =
     instance.customizeColumnsProps;
-  if (isModalOpen) {
+  if (isTearsheetOpen) {
```

Comment:
Do we actually need to check this state here? I think this is what’s causing the broken animation.
CC @AlexanderMelox @davidmenendez